service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -139,72 +142,82 @@ else if (e1.getMessage().contains(Integer.toString(HttpStatus.EXPECTATION_FAILED
     private JsonObject erLookup(final JsonObject bodyJson, Boolean failIfMultipleFound, Boolean failIfNoneFound,
                final Boolean lookupInExternalERs, final int lookupLimit)
             throws REMGenerateException {
-
         // Checking ER lookup enabled or not
         if (erlookupConfig.getEventRepositoryEnabled() && bodyJson.toString().contains("%lookup%")) {
-            JsonArray lookupLinks = bodyJson.get("eventParams").getAsJsonObject().get("links").getAsJsonArray();
-            JsonArray links = new JsonArray();
-            for (int i = 0; i < lookupLinks.size(); i++) {
-                if (lookupLinks.get(i).toString().contains("%lookup%")) {
-                    String[] ids = null;
+            // Checking ER lookup limit
+            if(lookupLimit > 0) {
```

Comment:
can we not have validation in Spring page to set the value or override the value to 1 if -ve or 0 is set.
I would rather have that kind of implementation rather that throwing exception to the user.

Reply:
Actually, by @RequestParam the defaultValue is set to 1. But in spring page if we want to  override the value to 1 if -ve or 0 need to check how to set the overridden value. If anyone has idea on it, It could be helpful.